cedar-policy-core/src/ast/expr.rs

```diff
@@ -321,6 +321,13 @@ impl Expr {
         ExprBuilder::new().ite(test_expr, then_expr, else_expr)
     }
 
+    /// Create a ternary (if-then-else) `Expr`.
+    /// Takes `Arc`s instead of owned `Expr`s
+    /// `test_expr` must evaluate to a Bool type
+    pub fn ite_arc(test_expr: Arc<Expr>, then_expr: Arc<Expr>, else_expr: Arc<Expr>) -> Self {
```

Comment:
Nit: can we make it `pub(crate)`?

Reply:
Rest of the functions are not `pub(crate)`, why this one?

Reply:
I think the signature of this function is not idiomatic as public functions. I was thinking another function as well but just commented on this one.